jac-uk/admin
# #2079 Eligibility Checkbox Groups Validate Error

## What's included?
Updated to use fix in jac-kit. Altered checkbox group variables to be empty arrays by default instead of null.

Closes #2079 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

- Create a new exercise
- Go to the Eligibility tab and fill in the form fields and save with no errors
- Click back into the Eligibility tab
- Click the Update Eligibility Information link
- Scroll to the bottom and press the Save And Continue button
- Ensure there are no errors for the Authorisation field
- Deselect everything in the Authorisation field and press Save And Continue and ensure an error displays telling the user they need to fill in this field

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
